packages/testkit-backend/package.json

```diff
@@ -24,10 +30,17 @@
   },
   "homepage": "https://github.com/neo4j/neo4j-javascript-driver#readme",
   "dependencies": {
+    "neo4j-driver": "4.4.0-dev",
     "neo4j-driver-lite": "4.4.0-dev",
-    "neo4j-driver": "4.4.0-dev"
+    "node-static": "^0.7.11",
+    "ws": "^8.2.3"
   },
   "devDependencies": {
-    "esm": "^3.2.25"
+    "@rollup/plugin-commonjs": "^21.0.1",
+    "@rollup/plugin-node-resolve": "^13.0.6",
+    "esm": "^3.2.25",
+    "rollup": "^2.59.0",
```

Comment:
The `public/index.js` is generated using `rollup` during the `npm run build`. `Rollup` is not need anymore when we run the server, since everything is already built. 

It's pretty much what's happening with the driver, but the driver uses `browserify`, `babel` and `gulp` for building the `lib` and the bundled version which could be imported in browsers. This libraries are used as building tools in the development process, but they are not need in runtime.